### PR TITLE
Vb

### PR DESCRIPTION
**Pull request by gidden**
move virtual box instructions to website

### CODE REVIEW

**Comment by gidden** [comment]
ping @opotowsky, @mbmcgarry, @gonuke, etc.

File: source/user/virtualbox.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+#. Install `VirtualBox <https://www.virtualbox.org/>`_
+#. Download `cyclus_user_environment.ova
+   <http://cnergdata.engr.wisc.edu/cyclus/virtual-box/current/cyclus_user_environment.ova>`_
+#. Open the .ova file using VirtualBox (File->Import Appliance).
+#. Login with the username ``ubuntu`` and pw ``reverse``
+#. Open a terminal and type ``conda update``
+
+Congratulations! You now have the same user environment as someone who installed
+cyclus on an Ubuntu Linux machine.
```
Comment (mbmcgarry) [inline source/user/virtualbox.rst]: |Cyclus|

File: source/user/virtualbox.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
+
+Installing a Cyclus VirtualBox
+==============================
+
+A native Windows installation is not currently explicitly supported. However, a
+Virtual Machine image can be downloaded which includes a pre-installation of all
```
Comment (mbmcgarry) [inline source/user/virtualbox.rst]: which->that

**Comment by mbmcgarry** [comment]
Not blocking, but related - Do we expect people attending the tutorials to have already prepared their computers? If so then links to these pages  should be put explicitly on the tutorial pages as well (tutorials/2015-ans-users.html and tutorials/2015-ans-archdev.html)

**Comment by gidden** [comment]
Correct me if I'm wrong, @gonuke, but we expect people to come to the *dev*
tutorial with cyclus already installed, but we expect to have a cyclist
installation exercise for the *user* tutorial?

On Wed, Jun 3, 2015 at 9:17 AM, mbmcgarry <notifications@github.com> wrote:

> Not blocking, but related - Do we expect people attending the tutorials to
> have already prepared their computers? If so then links to these pages
> should be put explicitly on the tutorial pages as well
> (tutorials/2015-ans-users.html and tutorials/2015-ans-archdev.html)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cyclus.github.com/pull/156#issuecomment-108447014>
> .
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192

**Comment by gidden** [comment]
But yes, in general both of those sites need to be updated.

On Wed, Jun 3, 2015 at 10:21 AM, Matthew Gidden <gidden@wisc.edu> wrote:

> Correct me if I'm wrong, @gonuke, but we expect people to come to the
> *dev* tutorial with cyclus already installed, but we expect to have a
> cyclist installation exercise for the *user* tutorial?
>
> On Wed, Jun 3, 2015 at 9:17 AM, mbmcgarry <notifications@github.com>
> wrote:
>
>> Not blocking, but related - Do we expect people attending the tutorials
>> to have already prepared their computers? If so then links to these pages
>> should be put explicitly on the tutorial pages as well
>> (tutorials/2015-ans-users.html and tutorials/2015-ans-archdev.html)
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/cyclus/cyclus.github.com/pull/156#issuecomment-108447014>
>> .
>>
>
>
>
> --
> Matthew Gidden, Ph.D.
> Postdoctoral Associate, Nuclear Engineering
> The University of Wisconsin -- Madison
> Ph. 225.892.3192
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192

**Comment by mbmcgarry** [comment]
Thanks @gidden !

**Pull request merged**